hack/test/integration.sh

```diff
@@ -287,10 +287,15 @@ ${ARTIFACTS}/talosctl cluster create \
   --name test-1 \
   --cidr=172.20.0.0/24 \
   --no-masquerade-cidrs=172.21.0.0/24,172.22.0.0/24 \
-  --input-dir="${PARTIAL_CONFIG_DIR}" \
+  --skip-injecting-config \
+  --wait=false \
   --vmlinuz-path="https://factory.talos.dev/image/${SCHEMATIC_ID}/v${TALOS_VERSION}/kernel-amd64" \
   --initrd-path="https://factory.talos.dev/image/${SCHEMATIC_ID}/v${TALOS_VERSION}/initramfs-amd64.xz"
 
+${ARTIFACTS}/talosctl apply-config --insecure --nodes 172.20.0.2 --file "${PARTIAL_CONFIG_DIR}/controlplane.yaml"
```

Comment:
I'd say better to do in a loop, looping for the total number of machines (extract machine counts to variables).